update open data sharing guidelines (#1478)
agrc/gis.utah.gov: gis-data-sharing/open-data/index.md

  display_name: Michael M. Foulger
  email: michaelfoulger@utah.gov
date: 2019-11-12 13:40:06
title: Sharing Data through the SGID Open Data Site
categories: []
---

> ![Open-Data-Homepage]({% link images/od_homepage.png %}){: style="width:50%; margin: 0 auto; display: block;" .outline }

Reviewer: ```suggestion
![Open-Data-Homepage]({% link images/od_homepage.png %}){: .flex .flex--center .outline }
```

> Does your department or division have authoritative datasets hosted in your own ArcGIS Online organization? Do you want to make them easy to find by anyone searching for Utah GIS Data? Share them on the [SGID Open Data website!](https://opendata.gis.utah.gov/) This makes them them easily discoverable while giving you full control over when they are updated.

Reviewer 2: ```suggestion
Does your department or division have authoritative datasets hosted in your own ArcGIS Online organization that are important to broadcast publicly? Do you want to make them easy to find by anyone searching for Utah GIS Data? Share them on the [SGID Open Data website!](https://opendata.gis.utah.gov/) This makes them them easily discoverable while giving you full control over when they are updated.
```

Author: I think I see where you're going with this, but I think the next sentence covers the public availability bit. I'm aiming for a logical progression: Data in a separate AGOL org? Check. Want to make it easy to find? Check. Then, share with open data.

## Adding Your Data to Open Data

All you need to do is **share your ArcGIS Online items with the appropriate SGID groups**. Items shared with these ArcGIS Online groups are automatically added to the SGID Open Data site.

> First, you have to be a member of the SGID groups your datasets best fit in. Navigate to AGRC's [groups page](https://utah.maps.arcgis.com/home/groups.html) and click on the appropriate group. Then click on "Join Group" to request access. We'll verify your request and add you to the group.

Reviewer 2: ```suggestion
First, you have to be a member of the SGID groups that are the best fit for your datasets. Navigate to AGRC's [groups page](https://utah.maps.arcgis.com/home/groups.html) and click on the appropriate group. Then click on "Join Group" to request access. We'll verify your request and add you to the group.
```

> Now that you're a member of the groups, share each item to the appropriate group. You also need to make sure the item is shared with "Everyone (public)" as well. Otherwise, it will only be visible to you (or other members of your organization if shared to the organization) in Open Data.

Reviewer 2: ```suggestion
Now that you're a member of the groups, share each item to the appropriate group. You also need to make sure the item is shared with "Everyone (public)". Otherwise, it will only be visible to you (or other members of your organization if shared to the organization) in Open Data.
```

## Presentation Tips

> ![Open-Data-Card]({% link images/od_card_anno.png %}){: style="margin: 0 auto; display: block; width=75%" }

Reviewer: ```suggestion
![Open-Data-Card]({% link images/od_card_anno.png %}){: .flex .flex--center }
```

> - The **title** is first place people look for information. Because items in Open Data are also ArcGIS Online items that are searchable by anyone in the world, we have added "Utah" to the beginning of every title of our public ArcGIS Online items to provide better context.

Reviewer 2: ```suggestion
- The **title** is the first place people look for information. Because items in Open Data are also ArcGIS Online items that are searchable by anyone in the world, we have added "Utah" to the beginning of every title of our public ArcGIS Online items to provide better context.
```

- A **user name** that includes the department/division's name or function helps users know where the dataset is coming from. For example, all the datasets we share are owned by the `UtahAGRC` user.
- Open Data displays the text from the `Description` field of the source ArcGIS Online item for the **description**. You shouldn't count on any information in the ArcGIS Online `Summary` field to show up unless it's repeated in the description.
- To help with searching, we include the "AGRC" and "SGID" **tags** on all our items while removing any tags that are repeated in the item's title. Please refer to Esri's [post about using tags effectively](https://www.esri.com/arcgis-blog/products/arcgis-online/data-management/using-tags-effectively/) for more ideas and information.

> - The "Last Updated" **date** is based on the last time the _ArcGIS Online item_ was changed, not the _data itself_. Things like updating the item's description or changing the thumbnail may update this date. For some frequently-updated datasets, we include the last update date in the description text or refer to the layer's description page on [gis.utah.gov/data]({% link data/index.html %})).

Reviewer: **chore**: remove duplicate paren
```suggestion
- The "Last Updated" **date** is based on the last time the _ArcGIS Online item_ was changed, not the _data itself_. Things like updating the item's description or changing the thumbnail may update this date. For some frequently-updated datasets, we include the last update date in the description text or refer to the layer's description page on [gis.utah.gov/data]({% link data/index.html %}).
```

> - The "Last Updated" **date** is based on the last time the *ArcGIS Online item* was changed, not the *data itself*. Things like updating the item's description or changing the thumbnail may update this date. For some frequently-updated datasets, we include the last update date in the description text or refer to the layer's description page on [gis.utah.gov/data](https://gis.utah.gov/data).

Author: ... I totally blanked that the data pages were an internal link. Thanks for catching that. 

I missed the update to the readme with all the hints and conventions. I'll take a look at that and review the page with that.

> - The "Last Updated" **date** is based on the last time the *ArcGIS Online item* was changed, not the *data itself*. Things like updating the item's description or changing the thumbnail may update this date. For some frequently-updated datasets, we include the last update date in the description text or refer to the layer's description page on [gis.utah.gov/data](https://gis.utah.gov/data).

Reviewer: No problem. That content was there before you were hired. I'll try to do a better job of showing it to folks. Maybe a contributors guide would be helpful.

- Keeping your **data schema** constant is a real help to your users. Some will be using your feature services directly in their webmaps rather than self-hosting a copy of your data and trying to keep it up-to-date. Removing fields or changing their types can break popups, search widgets, and other functionality. Adding new fields is less dangerous.
- By default, users can't download spatial datasets from ArcGIS Online/Open Data as **File Geodatabases**. To enable this, go to the item's page in ArcGIS Online and click on the "Settings" tab. Scroll to the bottom and check the box to "Allow others to export data to different formats".
  - NOTE: This will also allow people to export/copy some or all of the features if they add the dataset as a layer in ArcGIS Pro.

> ## Taming the Sources List

Reviewer: **thought**: this whole section seems like it will be more of an implementation detail once the sources page is created and we have a better way to deal with finding sources. While I agree that folks should be careful with their sources, I wonder if this is too much information for a small reward.

Author: The in-depth description is driven by a conversation I had with Zach and trying to explain to another agency why they weren't showing up in the sources list they way they thought they should. It felt like one of those things that it's better to write once and refer people to rather than repeat multiple times in individual emails.

Reviewer: That is good to know. I wouldn't think agencies would have noticed.

So where do the source values come from? Esri explains it [in this FAQ](https://doc.arcgis.com/en/hub/get-started/frequently-asked-questions.htm#GUID-9843B713-46D2-4938-A961-EC0CD81AE410). The main take away is that it first searches the **ArcGIS Online item's metadata** for an organization name. If it doesn't find anything there, it falls back to your **ArcGIS organization name**.

If you'd like to customize the name displayed in the sources list, we suggest setting it in the `Resource > Citation > Contact > Organization Name` metadata field. Otherwise, make sure the metadata contact fields are empty and let it pick up your ArcGIS Online organization name. We use this second way to show "Utah Automated Geographic Reference Center (AGRC)" as the source for all our datasets.

To change the organization name in the ArcGIS Online item's metadata, you need to make sure your ArcGIS Online organization has [metadata enabled](https://doc.arcgis.com/en/arcgis-online/administer/configure-details.htm#ESRI_SECTION1_7FE1F060E03046E692BB36E5F6E3B2AE). Then you can use the Metadata editor on the ArcGIS Online item page to set either field:

> ![Citation-Contact]({% link images/od_citation_contact_orgname.png %}){: style="width:400px; margin: 10px" .outline } ![Contact]({% link images/od_contact_orgname.png %}){: style="width:400px; margin: 10px" .outline }

Reviewer: <div class="flex flex--center" markdown="1">
![Citation-Contact]({% link images/od_citation_contact_orgname.png %}){: .outline } ![Contact]({% link images/od_contact_orgname.png %}){: .outline }
</div>

> Depending on how you upload or overwrite your ArcGIS Online data, it may also overwrite the item's metadata with info from the source **feature class metadata**. We suggest you make any changes in both the feature class and ArcGIS Online metadata to avoid problems in the futre. In the FGDC CSDGM style, the `Resource > Citation > Contact > Organization Name` field is in `Overview > Citation Contacts`. The `Resource > Contact > Organization Name` field is in `Resource > Points of Contact`.

Reviewer: ```suggestion
Depending on how you upload or overwrite your ArcGIS Online data, it may also overwrite the item's metadata with info from the source **feature class metadata**. We suggest you make any changes in both the feature class and ArcGIS Online metadata to avoid problems in the future. In the FGDC CSDGM style, the `Resource > Citation > Contact > Organization Name` field is in `Overview > Citation Contacts`. The `Resource > Contact > Organization Name` field is in `Resource > Points of Contact`.
```
